cocotb/share/lib/embed/gpi_embed.c

```diff
@@ -392,20 +458,17 @@ void embed_sim_event(gpi_event_t level, const char *msg)
         to_python();
         gstate = PyGILState_Ensure();
 
-        PyObject *fArgs = PyTuple_New(2);
-        PyTuple_SetItem(fArgs, 0, PyInt_FromLong(level));
-
-        if (msg != NULL)
-            PyTuple_SetItem(fArgs, 1, PyString_FromString(msg));
-        else
-            PyTuple_SetItem(fArgs, 1, PyString_FromString("No message provided"));
-        PyObject *pValue = PyObject_CallObject(pEventFn, fArgs);
-        if (!pValue) {
+#if PY_MAJOR_VERSION >= 3
+        PyObject* pValue = PyObject_CallFunction(pEventFn, "ls", level, (msg != NULL) ? msg : "No message provided");
+#else
+        static char format[] = "ls";
+        PyObject* pValue = PyObject_CallFunction(pEventFn, format, level, (msg != NULL) ? msg : "No message provided");
```

Comment:
Probably worth a comment here, maybe just `// workaround for bpo-9369, fixed in 3.4`

Reply:
Added comment.